assets/src/editor/events/dates-and-times/editor-date/edit-form/use-date-entity-input-config.js

```diff
@@ -64,6 +68,17 @@ const useDateEntityInputConfig = ( dateEntity ) => useMemo( () => {
 						! isNaN( newDate.getTime() )
 					) {
 						dateEntity.start = DateTime.fromJSDate( newDate );
+
+						const endDate = dateEntity.end.toJSDate();
+
+						if ( endDate - newDate < 0 ) {
+							// add 2 days to the end date.
+							endDate.setDate( endDate.getDate() + 2 );
+
+							dateEntity.end = DateTime.fromJSDate( endDate );
+
+							updateField( `${ prefix }-end`, endDate.toISOString() );
```

Comment:
can we plz remove the extra empty lines?

Reply:
Done :)